warehouse-management-service/db/migrations/000001_create_warehouse_table.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS warehouse;
```

Comment:
Is the omission of newlines here deliberate?

Reply:
It isn't deliberate, it seems to be the default of the SQL formatting tool I'm using

Reply:
Check if you're able to tweak this in that case, its an odd default. Conventionally, unix utils all expect text files to end in newlines. You might run into issues if these files ever need to processed by these things, for example in a CI pipeline or any other tool that uses these utils under the hood.